.gitlab-ci.yml

```
@@ -89,3 +89,75 @@ deploy_to_reliability_env:
     UPSTREAM_PACKAGE_JOB: build
     UPSTREAM_PROJECT_NAME: $CI_PROJECT_NAME
     FORCE_TRIGGER: $DEPLOY_TO_REL_ENV
+
+deploy_to_docker_registries:
+  stage: deploy
+  rules:
+    - if: '$POPULATE_CACHE'
+      when: never
+    - if: '$CI_COMMIT_TAG =~ /^v.*/'
```

I know I'm paranoid for nothing, but I'd rather have a stricter regex in case we decide to add a "verYBeautifulVersion" tag one day :)

Something like this (I'm the work person ever in regex, so please triple check)

```suggestion
    - if: '$CI_COMMIT_TAG =~ /^v[0-9]*\.[0-9]*\.[0-9]*/'
```

---

It's possible we may want to publish a prerelease tags so I think we could include it? I'll make the change as Andrew suggested

---

Done in e87558d547e2ab76b2070f58794614720a66d33c